containers/kubernetes/reference-content/kubernetes-load-balancer.mdx

```diff
@@ -167,6 +167,23 @@ From podA, a curl to `https://serviceB.example.com` will result in an SSL error.
 
 To force the Load Balancer to handle all requests using the SSL offload, enable the [use hostname annotations](https://github.com/scaleway/scaleway-cloud-controller-manager/blob/master/docs/loadbalancer-annotations.md#servicebetakubernetesioscw-loadbalancer-use-hostname) on your Load Balancer service.
 
+### Load Balancer TCP Proxy and Nginx Ingress error 400
+
+If your Load Balancer is configured using TCP Proxy and and you are using an Nginx Ingress, you might encounter an http 400 error.
+
+The Nginx Ingress must be configured with [use-proxy-protocol](https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/configmap/#use-proxy-protocol) to accept TCP Proxy requests.
+
+#### Nginx Ingress Easy Deploy
+
+This configuration is the default one when using the [Easy Deploy](/containers/kubernetes/how-to/enable-easy-deploy) Nginx Ingress.
```

Comment:
Proxy protocol on the loadbalancer is not enabled by default on nginx ingress from easydeploy. adding config bellow alone (without corresponding annotation) would break the ingress-lb communication.

Reply:
I checked and you are right, by default there is no proxy.

What is better ? give the "complete" Easy Deploy configuration for having a LB TCP proxy that works with nginx (annotation + config), or delete this subSection ?